base/applications/rapps/gui.cpp

```diff
@@ -891,9 +890,8 @@ VOID MainWindowLoop(INT nShowCmd)
             if (Msg.message == WM_CHAR &&
                 Msg.wParam == VK_TAB)
             {
-                // Move backwards if shift is held down
-                int direction = (GetKeyState(VK_SHIFT) & 0x8000) ? -1 : 1;
-
+                /* Move backwards if shift is held down */
+                INT direction = (GetKeyState(VK_SHIFT) & 0x8000) ? -1 : 1;
```

Comment:
```suggestion
                INT direction = (GetKeyState(VK_SHIFT) < 0) ? -1 : 1;
```

Reply:
No, the `& 0x8000` is the actual test (test for bit flag)

Reply:
The return value of `GetKeyState` is a `SHORT` (a 16-bit signed integer). `0x8000` means its most significant bit (MSB).